java/src/jmri/jmrix/openlcb/swing/ClientActions.java

```diff
@@ -48,7 +51,7 @@ public void openCdiWindow(NodeID destNode, String description) {
             private boolean haveButtons = false;
             @Override
             public JButton handleReadButton(JButton button) {
-                readList.add(button);
+//                readList.add(button);
```

Comment:
@icklesteve @bobjacobsen 
I got a warning that the list `readList` is only added to so I have comment it out. Could we have a bug here? I don't use olcb so I cannot test this.

Reply:
Having managed to get the OLCB jar files viewable in NetBeans, I see the same thing in that readList is only added to.
I'd say safe to remove, although I also don't use OLCB so cannot test.
Steve.